docs/ssm-deployment-guide.md

```diff
@@ -240,27 +237,24 @@ Run SSM
 
 * **Stop SSM server**
    
-   The script `bin/stop-ssm.sh` is used to stop SSM server. Remember to
-   use the exact same options to `stop-ssm.sh` script as to `start-ssm.sh` script.
-   For example, for starting Smart Server, you use
+   The script `bin/stop-ssm.sh` is used to stop SSM server.
 
-   `bin/start-ssm.sh --config ./YOUR_CONF_DIR`
+   `./bin/stop-ssm.sh`
 
-   Then, to stop Smart Server, you should use
+   `--config <config-dir>` can be used to specify where the config directory is.
```

Comment:
Better to remind users to use the same --config value  to execute `stop-ssm.sh` script as  `start-ssm.sh` script.